src/lib/io_environment.cpp

```diff
@@ -75,6 +77,11 @@ IOEnvironment& IOEnvironment::with_object_manager() {
     return get_instance();
 }
 
+IOEnvironment& IOEnvironment::with_grpc_client_workers(std::string const& name, uint32_t const num_workers) {
+    sisl::GrpcAsyncClientWorker::create_worker(name, num_workers);
+    return get_instance();
```

Comment:
What is all these `get_instance()` calls? Why not just `*this` like normal? This is a member function, *this *is* the singleton...

Reply:
You are right. I started returning get_instance when I first wrote this and am following it for continuity